pkg/apis/servicecatalog/types.go

```diff
@@ -49,9 +49,9 @@ type BrokerSpec struct {
 	AuthUsername string
 	AuthPassword string
 
-	// CF-specific; move to annotation
-	// CFGUID is the identity of this object for use with the CF SB API.
-	CFGUID string
+	// OSB-specific
+	// OSBGUID is the identity of this object for use with the OSB API.
+	OSBGUID string
```

Comment:
There is a UID, true, however:

1.  Not all of the OSB-specific things in the API are resources -- for example, plans have a GUID in the OSB API, but they are not resources in our API
2.  The UID of kubernetes objects is assigned by the API server, which isn't compatible with the OSB API where the GUID is reported back by the broker

So, at this point, I think we need to make a distinction between these fields.